src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -3,35 +3,35 @@
   (:require [re-frame.core :as re-frame]
             [clojure.string :as str]
             [status-im.components.react :as react]
-            [status-im.ui.screens.discover.styles :as st]
+            [status-im.ui.screens.discover.styles :as styles]
             [status-im.components.status-view.view :as view]
             [status-im.utils.gfycat.core :as gfycat]
             [status-im.utils.identicon :as identicon]
-            [status-im.components.chat-icon.screen :as ci]
-            [status-im.utils.platform :as platform]
-            [status-im.components.icons.vector-icons :as vi]
+            [status-im.components.chat-icon.screen :as chat-icon]
+            [status-im.components.icons.vector-icons :as vector-icons]
             [status-im.i18n :as i18n]))
 
 (defn title [label-kw action-kw action-fn active?]
-  [react/view st/title
-   [react/text {:style      (get-in platform/platform-specific [:component-styles :discover :subtitle])
-                :uppercase? (get-in platform/platform-specific [:discover :uppercase-subtitles?])
+  [react/view styles/title
+   [react/text {:style      styles/title-text
                 :font       :medium}
     (i18n/label label-kw)]
    [react/touchable-highlight {:on-press action-fn}
     [react/view {}
-     [react/text {:style (st/title-action-text active?)}
+     ;; NOTE(oskarth): text-transform to uppercase not supported as RN style
+     ;; https://github.com/facebook/react-native/issues/2088
+     [react/text {:style      (styles/title-action-text active?)
+                  :uppercase? (:uppercase? styles/subtitle-text-augment)}
```

Comment:
the workaround would be better in react/text component, might be worth a tech-debt ticket

Reply:
@yenda  Can we do this in a separate issue? This is improvement over current state of affairs